src/parser/commands.h

```diff
@@ -17,18 +17,19 @@
  * code.
  */
 
-#include "cvc5parser_public.h"
```

Comment:
Not sure what's going on here. `commands.h` is a private (internal) header? Why do we need to include `#include <cvc5/cvc5_parser.h>`?

Reply:
Currently since the commands inherit from the public `Command`.  

I think we actually should refactor Command so that these inherit from a `CommandInternal` which the public `Command` contains.  I'll try this now.